### ISSUE TITLE
Report window displays with hidden main area, and is thus confusing

### ISSUE THREAD

**Issue by kaimikael** [biolab/orange3#5195]
- [ ] **What's your use case?**

The widgets have a button named “Report”.  In the documentation its function is only explained as “Produce a report.”  Clicking the button brings up a dialogue, the purpose of which is not obvious.

I get confused by the button, and I’m a bit scared to just randomly press buttons in the dialogue to see what happens.

- [ ] **What's your proposed solution?**

I suggest there be a section in the documentation that explains how the Report button works (there might already be, but I haven’t found it by searching for it) and that the “Produce a report” bit in each widget doc link to that explanation.

- [ ] **Are there any alternative solutions?**
Alternatively, the popup dialogue could contain more hints to its purpose.

**Comment by ajdapretnar** [comment]
Probably it should be a part of the #5125.

**Comment by irgolic** [comment]
@janezd, could you write up some documentation on the report feature? (I remember you being very enthusiastic about it)

**Comment by markotoplak** [comment]
On a sidenote, I saw that for some users the Report windows opens so small, that the report part can not be seen. 

I just tried. This is what I get if I use report for the first time.

![image](https://user-images.githubusercontent.com/552182/104929969-dea81c80-59a4-11eb-99f2-60cd657fdeb5.png)

So this might be the the first issue to tackle here. The window should first appear with the right side shown, like this:

![image](https://user-images.githubusercontent.com/552182/104930178-18792300-59a5-11eb-9218-3e71503a25b2.png)
